### PR TITLE
made triple quotes work correctly

### PR DESCRIPTION
**Pull request by Graur**
Closes: #1885 

---

## PR-Codex overview
This PR makes various changes to multiple files. It fixes a string comparison test, adds a new test for triple-quoted strings, modifies a Rust code snippet, and updates a Java class. 

### Detailed summary
- Fixed a string comparison test in `eo-runtime/src/test/eo/org/eolang/string-tests.eo`.
- Added a new test for triple-quoted strings in `eo-runtime/src/test/eo/org/eolang/runtime-tests.eo`.
- Modified a Rust code snippet in `eo-maven-plugin/src/test/java/org/eolang/maven/BinarizeParseMojoTest.java`.
- Updated a Java class in `eo-parser/src/main/java/org/eolang/parser/XeListener.java`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by Graur** [comment]
@levBagryansky Please, have a look

File: eo-runtime/src/test/eo/org/eolang/runtime-tests.eo [hunk 1]
```diff
@@ -415,3 +415,12 @@
...
       "[Hello, Jeff]"
+
+[] > check-triple-quotes
+  assert-that > @
+    """
+    Hello
+
+    Hello
+    """
+    $.equal-to "Hello\n\nHello"
```
Comment (levBagryansky) [inline eo-runtime/src/test/eo/org/eolang/runtime-tests.eo:426]: Does `\n` always mean lineseparator in eo?

**Review by levBagryansky — CHANGES_REQUESTED** [review]
@Graur I like it! But I am not sure about `\n`

File: eo-runtime/src/test/eo/org/eolang/runtime-tests.eo (hunk 1 above)
Comment (Graur) [reply eo-runtime/src/test/eo/org/eolang/runtime-tests.eo:426]: @levBagryansky Both `\n` and `\r\n` are [valid](https://github.com/objectionary/eo/blob/master/eo-parser/src/main/antlr4/org/eolang/parser/Program.g4#L264-L266) linebreak

**Review by levBagryansky — APPROVED** [review]
@Graur ok, thanks! @yegor256 could you merge these chenges, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**